src/components/EnterpriseApp/data/enterpriseCurationReducer.js

```diff
@@ -47,6 +52,19 @@ function enterpriseCurationReducer(state, action) {
       return { ...state, enterpriseCuration: action.payload };
     case SET_FETCH_ERROR:
       return { ...state, fetchError: action.payload };
+    case SET_TOAST_TEXT: {
+      const existingHighlightSets = getHighlightSetsFromState(state);
+      const filteredHighlightSets = existingHighlightSets.filter(
+        highlightSet => highlightSet.uuid === action.payload,
+      );
```

Comment:
nit: considering we expect to only find 1 matching highlight set given the UUID passed in `action.payload`, should we opt for `existingHighlightSets.find` instead?

```jsx
const foundHighlightSet = existingHighlightSets.find(...);

return {
  ...state,
  enterpriseCuration: {
    ...state.enterpriseCuration,
    toastText: foundHighlightSet?.title,
  },
};
```